### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Tbaut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,63 @@
-# Fether
+# Parity Fether a fast and decentralized wallet based on a light-client
```
Comment (amaurymartiny) [inline README.md]: I would add a comma after Parity Fether

File: README.md [hunk 2]
```diff
@@ -1,21 +1,63 @@
...
+### Manually
+
+To start Parity Fether manually, just run
 
 ```bash
-git clone https://github.com/paritytech/fether
-yarn install
+$ fether
 ```
+You can pass specific flags to the underlying Parity Ethereum node such as:
```
Comment (axelchalon) [inline README.md]: FWIW this only holds if there is no already running node (i.e. if fether starts the node)

If Parity is already running on kovan, running `fether --chain mainnet` will still connect to Parity on Kovan. (Could be good in Fether to warn the user if the user passed some flags but there is an already running Parity instance and as such the flags are ignored cc @amaurymartiny)

Comment (Tbaut) [reply README.md]: +1 I will try to rephrase that to make it clear

File: README.md [hunk 3]
```diff
@@ -1,35 +1,92 @@
...
 
-https://github.com/paritytech/fether/releases
+[» Download the latest release «](https://github.com/paritytech/fether/releases)
 
-## Getting started
+
+### Join the chat!
+
+Get in touch with us on Gitter:
+[![Gitter: Parity](https://img.shields.io/badge/gitter-parity-4AB495.svg)](https://gitter.im/paritytech/parity)
```
Comment (5chdn) [inline README.md]: Shall we use a different Gitter for UI stuff?

Comment (Tbaut) [reply README.md]: yes, I think this would be good, and help separate things a little.

**Pull request merged**